modules/op-about-triggers.adoc

```diff
@@ -93,10 +93,12 @@ spec:
 <4> Parameters supplied by the `TriggerBinding` or `EventListerner` resources.
 <5> List of templates that specify the way resources must be created using the parameters received through the `TriggerBinding` or `EventListener` resources.
 
-* The `Trigger` resource connects the `TriggerBinding` and `TriggerTemplate` resources, and this `Trigger` resource is referenced in the `EventListener` specification.
+* The `Trigger` resource combines the `TriggerBinding` and `TriggerTemplate` resources, and the `interceptors` event processor. 
++
+Interceptors process all the events for a specific platform that runs before the TriggerBinding resource. They allow you to: perform payload filtering, verification, transformation, define and test trigger conditions, and implement other useful processing. Interceptors use Secret for verification. Once the event data passes through an interceptor, it then goes to the trigger before you pass the payload data to the trigger binding. You can also use an interceptor to modify the behavior of the associated trigger. This `Trigger` resource is referenced in the `EventListener` specification.
```

Comment:
My version:

Interceptors process all events for a specific platform that runs before the `TriggerBinding` resource. You can use interceptors to perform payload filtering, verification, transformation, defining and testing trigger conditions, and implement other useful processing. Interceptors use `Secret` for verification <<< of what? >>>. Once the event data passes through an interceptor, it goes to the trigger before you pass the payload data to the trigger binding. You can also use an interceptor to modify the behavior of the associated trigger referenced in the `EventListener` specification.

Reply:
Done.